### DISCUSSION TITLE
This Message, Ruffle will be support ActionScript 3 function?

### DISCUSSION THREAD

**Discussion by janilboo** [ruffle-rs/ruffle#9341] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/123889124/215339183-1e2bd2be-ae6f-4d81-949f-b5007f3f7ce8.png)

[Paint_style.zip](https://github.com/ruffle-rs/ruffle/files/10530243/Paint_style.zip)

**Comment by Lord-McSweeney** [comment, marked as answer]
Yes, it eventually will. In the meantime, you can click "ok" and continue, but the game may not work as intended.
